eng/Baseline.Designer.props

```diff
@@ -2,7 +2,7 @@
 <Project>
```

Comment:
@pranavkm Microsoft.AspNetCore.Mvc.Api.Analyzers is not listed in this file nor in Baseline.xml. Should it be?

Reply:
Sorry, I'm not sure what this file is. We published an update to the package already, didn't we? https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.Api.Analyzers/2.2.6

Reply:
This and Baseline.xml describe the package versions to use when the package is _not_ updated in a particular patch. Since we're not planning to build Microsoft.AspNetCore.Mvc.Api.Analyzers.csproj in this coming patch, should we ensure Mvc.Api.Analyzers.Test.csproj tests what we shipped last time?

Reply:
> should we ensure Mvc.Api.Analyzers.Test.csproj tests what we shipped last time?

If that's what we do for all other packages, that sounds like the reasonable thing to do.

Reply:
Done. See https://github.com/aspnet/AspNetCore/pull/12100/files#diff-96fa7dcb9c7726743838f28cc9629998R652 (an automatic addition) and https://github.com/aspnet/AspNetCore/pull/12100/files#diff-c763c910a1518ee2bfcbe97d4e86a6bcR77 (manually added but version updated correctly and automatically)